app/controllers/api/v3/scenarios_controller.rb

```diff
@@ -88,7 +88,7 @@ def create
 
         @scenario = Scenario.new
 
-        if scaler_attributes && ! attrs[:descale]
+        if scaler_attributes && ! attrs[:descale] && ! Area.derived?(attrs.fetch(:area_code))
```

Comment:
> @ChaelKruip does it seem okay to you to not be able to (re-)scale derived scenarios? I can imagine it being annoying to some users.

Hm. I think it should be possible still. People may want to open a scenario for a municipality and scale it to their neighbourhood or street etc. Right @AlexanderWirtz?